packages/mdc-toolbar/mdc-toolbar.scss

```diff
@@ -88,6 +88,92 @@ $mdc-toolbar-mobile-breakpoint: 599px;
   @include mdc-elevation(4);
 }
 
+.mdc-toolbar--fixed-lastrow-only {
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 1;
+
+  @include mdc-elevation(4);
+}
+
+.mdc-toolbar--flexible {
+  // stylelint-disable plugin/selector-bem-pattern
+  // bem linter is disabled because it cannot correctly reconcile mdc prefix for css variables.
+  --mdc-toolbar-ratio-to-extend-flexible: 4;
```

Comment:
I feel like this should use `$mdc-toolbar-ratio-to-extend-flexible`; Also, if this is something that can be changed by an end user, the sass variable should be made `!default`.